staging/src/k8s.io/apiserver/pkg/server/storage/storage_factory_test.go

```diff
@@ -185,3 +185,54 @@ func TestUpdateEtcdOverrides(t *testing.T) {
 
 	}
 }
+
+func TestConfigs(t *testing.T) {
+	exampleinstall.Install(scheme)
+	defaultEtcdLocation := []string{"http://127.0.0.1", "http://127.0.0.2", "http://127.0.0.3"}
```

Comment:
nit: `defaultEtcdLocations` (as there is more than 1)

Reply:
Done